### PR TITLE
feat: custom user agent

### PR DESCRIPTION
**Pull request by admon84**
### Description

Configurable user agent string as an environment variable.

This may be able to help cut down on captchas, if we match the user agent that our PC/browser is using.

### Testing

* Set the user agent string in .env config and ran the script.  I noticed it helped me stop getting frequent amazon captchas (for now, at least)

### New dependencies

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Here is a list of variables that you can use to customize your newly copied `.en
 
 | **Environment variable** | **Description** | **Notes** |
 |:---:|---|---|
+| `USER_AGENT` | Custom User-Agent header for HTTP requests; optional |
```
Comment (jef) [inline README.md:65]: Do you mind putting this in order?

Same with example?

File: README.md [hunk 2]
```diff
@@ -87,6 +88,8 @@ Here is a list of variables that you can use to customize your newly copied `.en
 
 > :point_right: If you have multi-factor authentication (MFA), you will need to create an [app password](https://myaccount.google.com/apppasswords) and use this instead of your Gmail password.
 
+> :point_right: You can find your computer's user agent by searching google for ["my user agent"](http://google.com/search?q=my+user+agent)
```
Comment (jef) [inline README.md:91]:
```suggestion
> :point_right: You can find your computer's user agent by [searching google for "my user agent"](http://google.com/search?q=my+user+agent)
```

**Review by jef — CHANGES_REQUESTED** [review]

**Comment by jef** [comment]
I like it!

File: README.md (hunk 1 above)
Comment (admon84) [reply README.md]: Done.  I didn't realize they were alphabetized.

**Review by jef — APPROVED** [review]

**Comment by jef** [comment]
Thanks for all the hard work @admon84!

**Pull request merged**